standard_reports/README.md

```diff
@@ -62,18 +62,18 @@ Optional fields in the manifest json are marked as '// optional'
   // name of dir within the version dir of the report which includes js wasm function constructors.
   "convert_data": "convert_data_js",
   // optional
-  // name of html header template of the report found within the src dir
+  // name of tera html header template of the report found within the src dir
   "header": "header.html",
   // optional
-  // name of html footer template of the report found within the src dir
+  // name of tera html footer template of the report found within the src dir
```

Comment:
You can't really have comments in `json`, which is why the formatter is screaming here. It might work if you change the code markup to `json5`, which does allow comments. Looks fine in VS Code when I do that, not sure about here.